lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1002,8 +1032,10 @@ func validateBumpFeeRequest(in *BumpFeeRequest) (
 func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 	op wire.OutPoint, currentHeight int32) (sweep.Params, bool, error) {
 
-	// Return an error if both deprecated and new fields are used.
-	feerate, immediate, err := validateBumpFeeRequest(in)
+	// Return an error if the bump fee request is invalid.
+	feerate, immediate, err := validateBumpFeeRequest(
+		in, w.cfg.FeeEstimator,
```

Comment:
IIUC this shouldn't even compile? 2 args vs 3

Reply:
hmm wdym, the function has only 2 params  ?

`func validateBumpFeeRequest(in *BumpFeeRequest, estimator chainfee.Estimator) (
	fn.Option[chainfee.SatPerKWeight], bool, error) {`

Reply:
Ah, I misread the siganture due to the way the line break was done here.